src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSessionDetailsPanel.tag

```diff
@@ -30,7 +30,7 @@
                 <div class="form-group">
                     <label class="col-sm-2 control-label">Closing time:</label>
                     <div class="col-sm-10">
-                        <p class="form-control-static">${feedbackSession.endTimeString}</p>
+                        <p class="form-control-static" id="end-time">${feedbackSession.endTimeString}</p>
```

Comment:
@whipermr5 Thanks for that, that was very helpful! Apologies for not picking that up myself, I am a little new to Java's Date and Calendar classes and as such was unsure what exactly everything was doing :p. Have read up on the docs now, and should, hopefully, not need as much hand-holding. Thanks again! 😄